amqp-protocol-binding.md

```diff
@@ -186,8 +174,32 @@ Examples:
 ##### 3.1.3.2 AMQP Application Property Values
 
 The value for each AMQP application property is constructed from the respective
-attribute's AMQP type representation, compliant with the [AMQP event
-format][amqp-format] specification.
+attribute's AMQP type representation.
+
+The CloudEvents type system MUST be mapped to AMQP types as follows, with
+exceptions noted below.
+
+| CloudEvents   | AMQP                        |
+| ------------- | --------------------------- |
+| Boolean       | [boolean][amqp-boolean]     |
+| Integer       | [long][amqp-long]           |
+| String        | [string][amqp-string]       |
+| Binary        | [binary][amqp-binary]       |
+| URI           | [string][amqp-string]       |
+| URI-reference | [string][amqp-string]       |
+| Timestamp     | [timestamp][amqp-timestamp] |
+
+A CloudEvents AMQP *binary* mode implementation MUST allow for attribute values
```

Comment:
That types be "convertible" is a property of an SDK,  not the protocol binding - the core spec already requires it. 
I'd say something like "all attribute values in an AMQP binary message MAY be represented using the native AMQP types above, or MAY be in canonical string form. An implementation MUST be able to interpret both forms on an incoming AMQP message. An implementation SHOULD use the native AMQP form on outgoing AMQP messages when it is efficient to do so, but MAY forward values as canonical strings. A robust implementation MAY further relax the requirements for incoming messages (for example accepting numeric types other than AMQP long) but MUST use only the prescribed AMQP types or canonical strings on outgoing messages.

Reply:
I like the proposed text and updated my proposal accordingly. Some people might argue with some of the MUST/SHOULD/MAY statements. I like how it is leaning towards the AMQP types without enforcing it.